Formula/mrboom.rb

```diff
@@ -0,0 +1,23 @@
+class Mrboom < Formula
+  desc "Mr.Boom is an 8 player Bomberman clone"
+  homepage "http://mrboom.mumblecore.org/"
+  url "https://github.com/Javanaise/mrboom-libretro/archive/3.9.tar.gz"
+  sha256 "166593ceed047464650b2de3c377c3cac1c9e3d807abe374e03b901f8ed6ab6e"
+
+  depends_on "cmake" => :build
+  depends_on "libmodplug"
+  depends_on "lzlib"
+  depends_on "minizip"
+  depends_on "sdl2"
+  depends_on "sdl2_mixer"
+
+  def install
+    system "make", "mrboom", "LIBSDL2=1"
+    system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man6}"
```

Comment:
needs to be with the relative path `MANDIR=share/man/man6`

Reply:
that's changed inside the Makefile
you think we should change this to
system "make", "install", "PREFIX=#{prefix}", "MANDIR=share/man/man6" ?

Reply:
yeah your MANDIR is defined as a relative path https://github.com/Javanaise/mrboom-libretro/blob/master/Makefile#L13

Reply:
it's changed there https://github.com/Javanaise/mrboom-libretro/blob/master/Makefile#L99

Reply:
Ah, then don't set it at all here. just PREFIX